api/app/serializers/spree/api/v2/platform/base_serializer.rb

```diff
@@ -3,6 +3,13 @@ module Api
     module V2
       module Platform
         class BaseSerializer < ::Spree::Api::V2::BaseSerializer
+          attribute :webhook_metadata, if: proc { |_record, params|
```

Comment:
@MatthewKennedy I think this should be in the upper node, for example structure

```js
{
  'event_id': '<webhook event.id>',
  'event_type': 'product.create',
  'event_created_at': '<webhook event.created_at>',
  'data':  {
    'id': '<product.id>',
    'type': 'product',
    'attributes': { ... },
    'relationships':  { ... }
  }
}
```

Reply:
@damianlegawiec i'll have to look into this.

Reply:
Does this `event_created_at` name refer to the column of `Spree::Webhooks::Event`? Currently the event is created after the request is made, we can't have the `id` and/or `created_at` value before that @damianlegawiec.

Reply:
@sebastian-palma we need to change it - because we and people intercepting these events will need the ID and date